Hogajama/hogajama-frontend/src/main/webapp/WEB-INF/keycloak.json

```diff
@@ -0,0 +1,9 @@
+{
+	"realm": "hogarama",
+	"auth-server-url": "${env.KEYCLOAK_AUTH_SERVER_URL}",
+	"ssl-required": "external",
+	"resource": "hogarama",
+	"credentials": {"secret":"12360720-0484-40da-9459-34ce53121677"},
+	"disable-trust-manager":true,
```

Comment:
Should disable-trust-manager be really set to true in production? See https://www.keycloak.org/docs/2.5/securing_apps/topics/oidc/java/java-adapter-config.html

Or is this still set to true because we do not have a configured truststore yet?